shared/output/cloud/language_v1/lib/google/cloud/language/v1/language_service/client.rb

```diff
@@ -152,8 +164,9 @@ def initialize
               credentials = @config.credentials
               # Use self-signed JWT if the endpoint is unchanged from default,
```

Comment:
should also be used if universe domain is not googleapis.com, whatever else is happening

Reply:
Yes, true, but the credential's universe is not necessarily known at this point in the client code. So this value is necessarily a universe-agnostic configuration—it means "should these credentials support SSJ authentication _if they are in the default universe_". (We could conceivably rename the keyword argument to clarify that, but I'm not going to bother at this point, especially since, as you note in your other comment, we may be able to remove this logic altogether.)

The logic enforcing SSJ in non-default universes is handled by the credential itself [here in googleauth](https://github.com/googleapis/google-auth-library-ruby/blob/main/lib/googleauth/service_account.rb#L102).

Reply:
:+1: